showcase/app/templates/page-components/form/super-select.hbs

```diff
@@ -1386,38 +1437,48 @@
   <Shw::Text::H3>Content</Shw::Text::H3>
 
   <Shw::Flex as |SF|>
-    <SF.Item @label="No arguments">
+    <SF.Item @label="No text arguments">
       <Shw::Outliner>
-        <Hds::Form::SuperSelect::AfterOptions />
+        <Hds::Form::SuperSelect::AfterOptions
+          @clearSelected={{this.noop}}
+          @showSelected={{this.noop}}
+          @showAll={{this.noop}}
+        />
       </Shw::Outliner>
     </SF.Item>
   </Shw::Flex>
 
   <Shw::Flex as |SF|>
     <SF.Item @label="with @showNoSelectedMessage=true">
       <Shw::Outliner>
-        <Hds::Form::SuperSelect::AfterOptions @showNoSelectedMessage={{true}} />
+        <Hds::Form::SuperSelect::AfterOptions
+          @showNoSelectedMessage={{true}}
+          @clearSelected={{this.noop}}
+          @showSelected={{this.noop}}
+          @showAll={{this.noop}}
+        />
       </Shw::Outliner>
     </SF.Item>
   </Shw::Flex>
 
   <Shw::Flex @direction="column" as |SF|>
     <SF.Item @label="with @content (text)" {{style max-width="500px"}}>
-      <Shw::Outliner>
-        <Hds::Form::SuperSelect::AfterOptions @content="Lorem ipsum dolor" />
-      </Shw::Outliner>
-    </SF.Item>
-    <SF.Item @label="with @content (generic)" {{style max-width="500px"}}>
       <Shw::Outliner>
         <Hds::Form::SuperSelect::AfterOptions
-          @content={{component "mock/components/form/super-select/generic-content"}}
```

Comment:
The diff for this change looks weird - but basically the after options component does not allow passing a component to the `@content` argument. 

I think this example is not needed because if someone wants to pass a custom component to the after options, they can use the `@afterOptionsComponent` argument instead (which is already covered by several examples on this page).